configure.ac

```diff
@@ -969,20 +964,13 @@ AC_CHECK_DECLS([CLOCK_MONOTONIC], [], [], [[
     #include <time.h>
 ]])
 
-# the above alone could allow using clock_gettime(CLOCK_MONOTONIC, ...)
-# in daemons/execd/execd_commands.c, but due to discovery of ftime causing
-# buildability problems in an out-of-the-box procedure with very recent glibc
-# snapshots (for which the above is meant to be a definitive solution)
-# coming very late prior to release, it is intentionally kept as opt-in,
-# with the line below (export CPPFLAGS="$CPPFLAGS -UPCMK_TIME_EMERGENCY_CGT"
-# or comment it out to revert this conservative delay of the roll out, e.g.,
-# when you hit said problems we eventually want to resolve, as already
-# materialized in Fedora Rawhide at this time), with the plan this
-# will only be applied in 2.0.3 release and will become opt-out since
-# (which hopefully explains the name of the macro as it will start to
-# make more sense then, and the continuity is important)
-CPPFLAGS="-DPCMK_TIME_EMERGENCY_CGT $CPPFLAGS"
-
+# the above alone will allow using clock_gettime(CLOCK_MONOTONIC, ...),
+# but in case there are any discrepancies found with the move onto that
+# where ftime(3) was originally used -- the callsites make do without
+# any such timestamp grabbing at this time -- so to revert that effect
+# (trigger such omission), line below can be uncommented for that intent
+# (alternatively, propagate equivalent variable assignment from outside)
+#CPPFLAGS="-DPCMK_TIME_EMERGENCY_CGT $CPPFLAGS"
```

Comment:
We can drop the emergency switch. The clock_gettime implementation will get several months of testing before the next release.

Reply:
The original idea as described in the former version of that comment

> will become opt-out since

was to keep this around for some time (perhaps one minor release)
would suffice